templates/template.yaml

```diff
@@ -107,12 +115,15 @@ Parameters:
   VpcSecurityGroups:
     Description: List of security groups to apply to Fargate tasks
     Type: CommaDelimitedList
+    Default: ""
   VpcSubnets:
     Description: List of subnets to deploy Fargate tasks in
```

Comment:
Should we specify here in the description that by leaving blank, a new VPC will be created? Just thinking of who doesn't read the docs